File: Python Basics/L12_while_loops_exercise/06_oscars.py
actor_name = input()
initial_points = float(input())
num_evaluators = int(input())

for _ in range(num_evaluators):
    evaluator_name = input()
    evaluator_points = float(input())

    points_from_evaluator = (len(evaluator_name) * evaluator_points) / 2
    initial_points += points_from_evaluator

    if initial_points > 1250.5:
        print(f"Congratulations, {actor_name} got a nominee for leading role with {initial_points:.1f}!")
        break
else:

    points_needed = 1250.5 - initial_points
    print(f"Sorry, {actor_name} you need {points_needed:.1f} more!")

File: CSharp Basics/04-For_Loops-exercise/06-Oscars.cs
using System;

namespace Oscars
{
    class Program
    {
        static void Main(string[] args)
        {
            string actorName = Console.ReadLine();
            double points = double.Parse(Console.ReadLine());
            int judgesCount = int.Parse(Console.ReadLine());

            for (int i = 1; i <= judgesCount; i++)
            {
                string judgeName = Console.ReadLine();
                double judgePoints = double.Parse(Console.ReadLine());

                double curPoints = judgeName.Length * judgePoints / 2;
                points += curPoints;

                if (points >= 1250.5)
                {
                    Console.WriteLine($"Congratulations, {actorName} got a nominee for leading role with {points:f1}!");
                    return;
                }
            }

            Console.WriteLine($"Sorry, {actorName} you need {(1250.5 - points):f1} more!");
        }
    }
}